src/Appwrite/Template/Template.php

```
@@ -63,7 +63,7 @@ public static function fromString(string $content): self
      *
      * @throws Exception
      */
-    public function render($minify = true): string
+    public function render($minify = true, $useContent = false): string
```

PHPMD (2.15.0)

Avoid unused parameters such as '$minify'. (Unused Code Rules)

(UnusedFormalParameter)